cosmos/x/evm/keeper/host.go

```diff
@@ -150,6 +149,6 @@ func (h *host) GetTxPoolPlugin() core.TxPoolPlugin {
 }
 
 // GetAllPlugins returns all the plugins.
-func (h *host) GetAllPlugins() []plugins.BaseCosmosPolaris {
-	return []plugins.BaseCosmosPolaris{h.bp, h.cp, h.gp, h.hp, h.pp, h.sp, h.txp}
+func (h *host) GetAllPlugins() []any {
```

Comment:
maybe we should make a base plugin that has a really stupid empty `IsPlugin()` function, so that at the very least we have an itnerface to conform to. I hate using `any`